slides/02_Setup.html

```diff
@@ -247,7 +247,16 @@ <h2>NPM scripts</h2>
             <p><span class="code">npm run dev</span></p>
             <p>
               Starts the app in dev mode with a hot reload server. The build
-              will fail if you have TypeScript errors.
+              will <strong>not</strong> run typescript checks.
+            </p>
+            <p>
+              Integrate typescript checking in the dev mode with the
+              <a
+                href="https://github.com/fi3ework/vite-plugin-checker"
+                target="_blank"
+                >vite-plugin-checker</a
+              >
+              or by running <span class="code">tsc --watch</span>.
```

Comment:
On the Vite docs, they recommend running `tsc --noEmit --watch`: https://vitejs.dev/guide/features.html#transpile-only -- emitting files by tsc is not needed as Vite dev mode does that already